src/algorithms/tracking/TrackProjector.cc

```diff
@@ -143,8 +143,13 @@ namespace eicrecon {
                 const float pathLength = static_cast<float>(trackstate.pathLength());
                 const float pathLengthError = 0;
 
-                // Store track point
+                uint64_t surface = trackstate.referenceSurface().geometryId().value();
+	        uint32_t system = 0;
+
+		// Store track point
                 track_segment.addToPoints({
+				                  surface,
+						  system,
```

Comment:
@jdbrice @tylerkutz This is the change you will want to make in main first, through a separate PR, to allow upgrading the data model in a backward and forward compatible way. These two extra fields in the initializer list are the issue. Changing to explicit setters is likely sufficient.